pkg/mcs/utils/util.go

```diff
@@ -135,10 +135,6 @@ func WaitAPIServiceReady(s server) error {
 		case <-ticker.C:
 		}
 	}
-	if err != nil {
-		log.Warn("failed to check api server ready", errs.ZapError(err))
-	}
```

Comment:
How about print warn log every 60 seconds or another better time?
I'm worried that all are debug logs won't be easy to find root case such as when the wrong api addr is set.

Reply:
make sense